conf/ocsci/mdr_workload.yaml

```diff
@@ -1,55 +1,125 @@
-    {name: "busybox-4", workload_dir: "mdr/subscriptions/busybox-app-4", pod_count: 2, pvc_count: 2},
-    {name: "busybox-5", workload_dir: "mdr/subscriptions/busybox-app-5", pod_count: 2, pvc_count: 2},
-  ]
-  dr_workload_appset: [
-    { name: "appset-busybox-1", workload_dir: "mdr/AppSet/appset-busybox-1",
-      dr_workload_app_placement_name: "busybox-1-placement",
-      dr_workload_app_pvc_selector: { 'appname': 'busybox_app1' }, pod_count: 2, pvc_count: 2
-    },
-    { name: "appset-busybox-2", workload_dir: "mdr/AppSet/appset-busybox-2",
-      dr_workload_app_placement_name: "busybox-2-placement",
-      dr_workload_app_pvc_selector: {'appname': 'busybox_app2'}, pod_count: 2, pvc_count: 2
-    },
-    { name: "appset-busybox-3", workload_dir: "mdr/AppSet/appset-busybox-3",
-      dr_workload_app_placement_name: "busybox-3-placement",
-      dr_workload_app_pvc_selector: {'appname': 'busybox_app3'}, pod_count: 2, pvc_count: 2
-    },
-    { name: "appset-busybox-4", workload_dir: "mdr/AppSet/appset-busybox-4",
-      dr_workload_app_placement_name: "busybox-4-placement",
-      dr_workload_app_pvc_selector: {'appname': 'busybox_app4'}, pod_count: 2, pvc_count: 2
-    },
-    { name: "appset-busybox-5", workload_dir: "mdr/AppSet/appset-busybox-5",
-      dr_workload_app_placement_name: "busybox-5-placement",
-      dr_workload_app_pvc_selector: {'appname': 'busybox_app5'}, pod_count: 2, pvc_count: 2
-    },
-  ]
-  dr_cnv_workload_appset_push: [
-    { name: "vm-appset-push-1", appset_model: "push",
-      workload_dir: "mdr/cnv-workload/appset/vm-appset-push-1",
-      dr_workload_app_placement_name: "vm-appset-push-1-placement", vm_name: "vm-workload-1",
-      vm_secret: "vm-secret-1", vm_username: "cirros",
-      dr_workload_app_pvc_selector: { 'appname': 'kubevirt' }, pod_count: 1, pvc_count: 1
-    },
-  ]
-  dr_cnv_workload_appset_pull: [
-    { name: "vm-appset-pull-1", appset_model: "pull",
-      workload_dir: "mdr/cnv-workload/appset/vm-appset-pull-1",
-      dr_workload_app_placement_name: "vm-appset-pull-1-placement", vm_name: "vm-workload-1",
-      vm_secret: "vm-secret-1", vm_username: "cirros",
-      dr_workload_app_pvc_selector: { 'appname': 'kubevirt' }, pod_count: 1, pvc_count: 1
-    },
-  ]
-  dr_cnv_workload_sub: [
-    { name: "vm-subscription-1", workload_dir: "mdr/cnv-workload/subscription/vm-subscription-1",
-      dr_workload_app_placement_name: "vm-sub-1-placement", vm_name: "vm-workload-1",
-      vm_secret: "vm-secret-1", vm_username: "cirros",
-      dr_workload_app_pvc_selector: { 'appname': 'kubevirt' }, pod_count: 1, pvc_count: 1
-    },
-  ]
+  dr_workload_subscription:
+    [
+      {
+        name: "busybox-1",
+        workload_dir: "mdr/subscriptions/busybox-app-1",
+        pod_count: 2,
+        pvc_count: 2,
+        is_placement: True,
+        dr_workload_app_pvc_selector: { "appname": "busybox_app1" },
+      },
+      {
+        name: "busybox-2",
```

Comment:
Examples from busybox-2 to busybox-5 may not work with recent changes done in dr_workload, how are we going to handle it?

Reply:
busybox-2 to busybox-5 will work as I have added is_placement: True.
Based on this condition workload will deploy as placement or placementrule